testem.js

```diff
@@ -3,20 +3,23 @@
 /* eslint-env node */
 
 module.exports = {
-    'test_page': 'tests/index.html?hidepassed',
-    'disable_watching': true,
-    'launch_in_ci': [
-        'Chrome'
-    ],
-    'launch_in_dev': [
-        'Chrome'
-    ],
-    'browser_args': {
-        'Chrome': [
-            '--disable-gpu',
-            '--headless',
-            '--remote-debugging-port=9222',
-            '--window-size=1440,900'
-        ],
+  test_page: 'tests/index.html?hidepassed',
+  disable_watching: true,
+  launch_in_ci: [
+    'Chrome'
+  ],
+  launch_in_dev: [
+    'Chrome'
+  ],
+  browser_args: {
+    Chrome: {
+      mode: 'ci',
+      args: [
+        '--disable-gpu',
+        '--headless',
+        '--remote-debugging-port=9222',
+        '--window-size=1440,900'
+      ]
     }
+  }
```

Comment:
tabsize for this file changed. we should try to stay consistent with the other files and use 4 spaces here too.